src/libasr/pass/replace_symbolic.cpp

```diff
@@ -842,6 +842,11 @@ class ReplaceSymbolicVisitor : public PassUtils::PassVisitor<ReplaceSymbolicVisi
                         x.base.base.loc, list_item->m_a, list_item->m_pos,
                         ASRUtils::TYPE(ASR::make_CPtr_t(al, x.base.base.loc)), nullptr));
                     print_tmp.push_back(basic_str(x.base.base.loc, value));
+                } else {
+                    ASR::expr_t *value = ASRUtils::EXPR(ASR::make_ListItem_t(al,
+                        x.base.base.loc, list_item->m_a, list_item->m_pos,
+                        ASRUtils::expr_type(val), nullptr));
+                    print_tmp.push_back(value);
```

Comment:
Let's simply push_back `val`.
```suggestion
                    print_tmp.push_back(val);
```